sdk/python/kfp/dsl/databricks/_cluster_op.py

```diff
@@ -0,0 +1,133 @@
+from .._resource_op import ResourceOp
+
+# Azure Databricks Operator and Golang SDK for Databricks types:
```

Comment:
it wasn't clear to me what these comments are used for
I looked through the pre-existing DSL classes and couldn't see anything similar
why are these here?

Reply:
I added them there for my reference on the expected specs, the status, behaviour of the API, success conditions, etc.

Reply:
I wonder if this would be better outside of the code, and in docs (even if it's just a README)

Reply:
I removed them.